packages/api-reference/src/components/MarkdownRenderer/MarkdownRenderer.vue

```diff
@@ -322,7 +324,7 @@ onServerPrefetch(async () => await sleep(1))
 
 <style lang="postcss">
 .markdown {
-  pre code[class*='language-'] {
+  pre code.hljs {
```

Comment:
```suggestion
  pre code[class*='language-'], pre code.hljs {
```
should have just left both classes tbh in case we go back

Reply:
but I guess we can always just come back to this PR and steal the code